columnar/src/column_values/u64_based/bitpacked.rs

```diff
@@ -16,6 +18,35 @@ pub struct BitpackedReader {
     stats: ColumnStats,
 }
 
+#[inline(always)]
+const fn div_ceil(n: u64, q: NonZeroU64) -> u64 {
+    // copied from unstable rust standard library.
+    let d = n / q.get();
+    let r = n % q.get();
+    if r > 0 {
+        d + 1
+    } else {
+        d
+    }
+}
+
+fn transform_range_before_linear_transformation(
+    stats: &ColumnStats,
+    range: RangeInclusive<u64>,
+) -> Option<RangeInclusive<u64>> {
+    if stats.min_value > *range.end() {
+        return None;
+    }
+    if stats.max_value < *range.start() {
+        return None;
+    }
+    let shifted_range =
```

Comment:
```suggestion
    if *range.end() < *range.start()  {
        return None;
    }
    let shifted_range =
```

Reply:
I added an equivalent condition at the beginning of the function.

Reply:
I think this condition needs to be added for correctness